apps/site/test/site_web/controllers/trip_plan_controller_test.exs

```diff
@@ -752,6 +752,14 @@ defmodule SiteWeb.TripPlanControllerTest do
       assert conn.assigns.query.to.name == "Geocoded Boston Common"
     end
 
+    test "uses expected values when addres is formatted latitutde,longitude,stopName" do
```

Comment:
this line needs to be edited to pass in the `conn` variable (see following test that happens to do this)

Reply:
added the conn variable and removed some excess spaces. This path doesn't make a query so should it be tested differently?